tests/testthat/test-retriever_v1_0_0_backward.R

```diff
@@ -0,0 +1,158 @@
+context("regression tests version 1.0.0")
+
+suppressPackageStartupMessages(require(reticulate))
+suppressPackageStartupMessages(require(DBI))
+# suppressPackageStartupMessages(require(RMariaDB))
+suppressPackageStartupMessages(require(RPostgreSQL))
+suppressPackageStartupMessages(require(RSQLite))
+
+
+# Set passwords and host names deppending on test environment
+os_password = ""
+pgdb = "localhost"
+mysqldb = "localhost"
+doker_or_travis = Sys.getenv("TRAVIS_OR_DOCKER")
+
+
+# Check if the environment variable "TRAVIS_OR_DOCKER" is set to "true"
+if (doker_or_travis %in% "doker_or_travis") {
```

Comment:
This is confusing for a couple of reasons:

1. Presumably this should be `==` not `%in%` since I don't think that `Sys.getenv("TRAVIS_OR_DOCKER" should ever return multiple items.
2. Checking if a variable is a string of the same name looks like it's probably a mistake. I understand that it isn't since `TRAVIS_OR_DOCKER` is set to `doker_or_travis`, but it's confusing. I recommend changing some names to clarify things. If it was me in `docker-compose.yml` I'd name the environmental variable `IN_DOCKER` and set it's value to `true`. Then in this test script I'd name the variable `in_docker` to match the system variable and test it against `== "true"`.